DQM/L1TMonitor/python/L1TStage2uGT_cfi.py

```diff
@@ -1,5 +1,4 @@
 import FWCore.ParameterSet.Config as cms
-
 from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
 l1tStage2uGT = DQMEDAnalyzer('L1TStage2uGT',
     l1tStage2uGtSource = cms.InputTag("gtStage2Digis"),    
```

Comment:
You need to add back the configuration for the trigger short list that you accidentally removed with this commit: https://github.com/cms-sw/cmssw/pull/22524/commits/99c195dcc61b8a09f9eb1c4e8831085f3622b952#diff-acd8aab9c38db89b4113dd4c8ee4402aR9 .
```
    lastBXInTrainAlgo = cms.untracked.string("L1_LastCollisionInTrain"),
    unprescaledAlgoShortList = cms.untracked.vstring("L1_SingleMu25","L1_SingleEG32","L1_DoubleMu_15_7","L1_SingleJet200","L1_HTT450er"),
    prescaledAlgoShortList = cms.untracked.vstring("L1_SingleMu3","L1_SingleEG15","L1_SingleJet90","L1_DoubleJet40er3p0","L1_HTT200er")
```